be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -319,9 +319,11 @@ void HdfsOrcScanner::update_counter() {
 
 #ifndef BE_TEST
     COUNTER_UPDATE(_scanner_params.parent->_raw_rows_counter, _stats.raw_rows_read);
+    COUNTER_UPDATE(_scanner_params.parent->_rows_read_counter, _stats.raw_rows_read);
```

Comment:
what is the difference between these two variables.

Reply:
The only difference is
- `_raw_rows_counter` is defined in `HDFSScanNode` class
- and `_rows_read_counter` is defined in `ScanNode` class

The another change is 

```
    COUNTER_UPDATE(_scanner_params.parent->_bytes_read_counter, _stats.bytes_read_from_disk);
    COUNTER_UPDATE(_scanner_params.parent->_bytes_read_from_disk_counter, _stats.bytes_read_from_disk);
```

I think the first one counter can be removed.  What do you think?

Reply:
AFAIK, `bytes_read` stands for data size in memory format in OLAPScanNode.
They are not the same thing.

Reply:
Got it. But it's quite hard to tell from the name `bytes_read`. It's quite straightforward to think this metrics is how many bytes  read from disk.